HardwareDefinitions/avnet_sk_rev2_files_for_SDK_installer/inc/hw/avnet_aesms_mt3620_rev2.h

```diff
@@ -0,0 +1,131 @@
+/* Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Can you please move this to avnet_mt3620_sk_rev2/inc/hw?

Reply:
Done